flytekit/tools/script_mode.py

```diff
@@ -183,6 +183,14 @@ def list_all_files(source_path: str, deref_symlinks, ignore_group: Optional[Igno
     return all_files
 
 
+def __drive_safe_commonpath(paths):
```

Comment:
Can you use one underscore here?

```suggestion
def _drive_safe_commonpath(paths):
```